charts/sysdig/templates/_helpers.tpl

```diff
@@ -127,4 +127,10 @@ NOTE: I don't like the error message! Too much information.
     {{- if hasKey .root.Values.sysdig.settings .setting }}{{ fail (printf "Value '%s' is also set via .sysdig.settings.%s'." .key .setting) }}{{- end -}}
     {{- $keyValue -}}
 {{- end -}}
+{{- end -}}
+
+{{- define "deploy-nia" -}}
```

Comment:
deploy-nia will be true if either .Values.nodeImageAnalyzer.deploy is true, or if .Values.nodeImageAnalyzer.settings.collectorEndpoint is set. 

The second condition covers the onboarding command for old on-prem versions, so if someone uses the on-prem onboarding helm command the old NIA is deployed (backwards compatibility)